WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewModel.kt

```diff
@@ -204,7 +201,9 @@ class StatsViewModel
             }
         }
 
-        if (statsSectionManager.getSelectedSection() == StatsSection.INSIGHTS) showInsightsUpdateAlert()
+        if (BuildConfig.IS_JETPACK_APP && statsSectionManager.getSelectedSection() == StatsSection.INSIGHTS) {
```

Comment:
Do we still need `statsSectionManager.getSelectedSection() == StatsSection.INSIGHTS` here? I guess we added that to show the update dialog on only the "INSIGHTS" tab. If so, we don't need it anymore.